src/bin/devmgr/src/main.rs

```diff
@@ -70,11 +95,31 @@ fn start_pcie(bus: Device) {
 }
 
 fn main() {
-    println!("[devmgr] starting device manager");
-    let device_root = twizzler_driver::device::get_bustree_root();
+    println!("[devmgr] starting device manager {:?}", args());
+    let id = args().into_iter().nth(1).unwrap().parse::<u128>().unwrap();
+    let obj = Object::<std::sync::atomic::AtomicU64>::init_id(
+        ObjID::new(id),
+        Protections::WRITE | Protections::READ,
+        ObjectInitFlags::empty(),
+    )
+    .unwrap();
+    let device_root = twizzler_driver::get_bustree_root();
     for device in device_root.children() {
         if device.is_bus() && device.bus_type() == BusType::Pcie {
             start_pcie(device);
         }
     }
+
+   // nvme_test::start();
```

Comment:
Just remove this or is this for a future test?

Reply:
Yes, this will be used in the next phase where I implement an nvme driver to dog-food these abstractions.